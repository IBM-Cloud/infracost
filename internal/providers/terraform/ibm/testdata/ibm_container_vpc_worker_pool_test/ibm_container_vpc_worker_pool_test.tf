

terraform {
  required_providers {
    ibm = {
      source  = "IBM-Cloud/ibm"
      version = "~> 1.12.0"
    }
  }
}

provider "ibm" {
  region = "us-south"
}

resource "ibm_container_vpc_worker_pool" "test_vpc_worker_pool" {
  cluster          = "my_vpc_cluster"
  worker_pool_name = "my_vpc_pool"
  flavor           = "bx2.4x16"
  vpc_id           = "6015365a-9d93-4bb4-8248-79ae0db2dc21"
  worker_count     = "2"

  zones {
    name      = "us-south-1"
    subnet_id = "015ffb8b-efb1-4c03-8757-29335a07493b"
  }
}
