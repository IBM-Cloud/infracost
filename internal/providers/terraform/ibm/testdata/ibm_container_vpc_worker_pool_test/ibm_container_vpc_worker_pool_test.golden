
 Name                                                                                        Monthly Qty  Unit   Monthly Cost 
                                                                                                                              
 ibm_container_vpc_worker_pool.test_vpc_worker_pool                                                                           
 └─ VPC Container Workpool flavor: (bx2.4x16) region: (us-south) workers x zones: (2) x (1)        1,460  hours       $376.94 
                                                                                                                              
 OVERALL TOTAL                                                                                                        $376.94 
──────────────────────────────────
1 cloud resource was detected:
∙ 1 was estimated